tests/test-cases/Group1-Docker-Commands/1-01-Docker-Info.robot

```diff
@@ -4,9 +4,34 @@ Resource  ../../resources/Util.robot
 Suite Setup  Install VIC Appliance To Test Server
 Suite Teardown  Cleanup VIC Appliance On Test Server
 
+*** Keywords ***
+Get resource pool CPU and mem values
+    [Arguments]  ${info}
+
+    ${cpuline}=  Get Lines Containing String  ${info}  CPUs:
+    ${memline}=  Get Lines Containing String  ${info}  Total Memory:
+    @{cpuline}=  Split String  ${cpuline}
+    Length Should Be  ${cpuline}  2
+    @{memline}=  Split String  ${memline}
+    Length Should Be  ${memline}  4
+    ${cpuval}=  Set Variable  @{cpuline}[1]
+    ${memunit}=  Set Variable  @{memline}[3]
+    ${memval}=  Set Variable  @{memline}[2]
+    # Since govc accepts a mem value only in MB, convert the value if necessary
+    ${memval}=  Run Keyword If  '${memunit}' == 'GiB'  Evaluate  int(round(${memval} * 1024))  ELSE  Evaluate  ${memval}
+
+    [Return]  ${cpuval}  ${memval}
+
+Set resource pool CPU and mem values
+    [Arguments]  ${cpuval}  ${memval}
+
+    ${rc}  ${output}=  Run And Return Rc And Output  govc pool.change -cpu.limit=${cpuval} %{TEST_RESOURCE}/${vch-name}
```

Comment:
fyi, I believe the path `%{TEST_RESOURCE}/${vch-name}` is specific to the ResourcePool on ESX.  In VC, the vApp itself is the resource pool and I believe the path would just be `./${vch-name}` (relative to the /dc-name/vm folder).